cts/cts-cli.in

```diff
@@ -1788,6 +1788,54 @@ EOF
 
 # Ensure all command output is in portable locale for comparison
 export LC_ALL="C"
+test_access_render() {
+    local TMPXML
+
+    # while the in-tree config would get picked normally by default,
+    # there's still a risk of artificial influence (pre-existing
+    # $PCMK_config_directory, user's local config), so eforce it here
+    if test -x "$SRCDIR/tools/cibadmin" && test -x "$SRCDIR/xml"; then
+        export PCMK_config_directory="$SRCDIR/xml/base"
+        echo "Using local configuration from: $PCMK_config_directory" >&2
+    fi
+
+    TMPXML=$(mktemp ${TMPDIR:-/tmp}/cts-cli.acls.xml.XXXXXXXXXX)
+    export CIB_shadow_dir="${shadow_dir}"
+
+    $VALGRIND_CMD crm_shadow --batch --force --create-empty $shadow 2>&1
+    export CIB_shadow=$shadow
+
+    cat <<EOF > "$TMPXML"
+    <acls>
+      <acl_role id="role-deny-acls">
+        <acl_permission id="deny-acls" kind="deny" xpath="/cib/configuration/acls"/>
+        <acl_permission id="read-rest" kind="read" xpath="/cib"/>
+      </acl_role>
+      <acl_target id="tony">
+        <role id="role-deny-acls"/>
+      </acl_target>
+    </acls>
+EOF
+
+    desc="Configure some ACLs"
+    cmd="cibadmin -M -o acls --xml-file $TMPXML"
+    test_assert $CRM_EX_OK
+
+    desc="Enable ACLs"
+    cmd="crm_attribute -n enable-acl -v true"
+    test_assert $CRM_EX_OK
+
+    desc="An instance of ACLs render (into color)"
+    cmd="cibadmin --force --access-render=color --user tony"
+    test_assert $CRM_EX_OK
+
+    desc="An instance of ACLs render (into full namespacing)"
+    cmd="cibadmin --force --access-render=ns-full --user tony"
+    test_assert $CRM_EX_OK
```

Comment:
To help keep the expected output cleaner, we can add "0" to the two test_assert lines for showing the ACLs.

The default is to show the CIB after the command, so we can ensure that any intended changes succeeded. But for the rendering commands we're not changing anything, just displaying, so "0" will tell test_assert not to bother showing the CIB after running the command.